frontend/src/components/buttons/DropdownSelect.styles.js

```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components';
+import caretDown from '../../images/caret-down.svg';
+
+export const DropdownWrapper = styled.div`
+    .ReactA11ySelect {
```

Comment:
One difference I realized that the font didn't match anymore so consider adding around here
`    font-family: 'Hack';`